var/spack/repos/builtin/packages/ca-certificates-mozilla/package.py

```diff
@@ -121,5 +123,7 @@ def setup_dependent_package(self, module, dep_spec):
     # Install the the pem file as share/cacert.pem
     def install(self, spec, prefix):
         share = join_path(prefix, "share")
-        mkdir(share)
+        # https://github.com/spack/spack/issues/32948
+        if not os.path.isdir(share):
+            mkdir(share)
```

Comment:
I believe `mkdirp()` from `llnl.util.filesystem` (implicitly imported from `spack.package`) does this check for you!

Reply:
@climbfuji if you do choose to use `mkdirp()` here I would still retain the link to #32948, that's a helpful comment for later readers.

Reply:
> Wouldn't it be better to put the sources in `<prefix>/.spack/src` or so? Then it's guaranteed not to clash.

Probably, but `spack install --source` doesn't take an argument, that's something for the spack developers to decide/change.

Reply:
> @climbfuji if you do choose to use `mkdirp()` here I would still retain the link to #32948, that's a helpful comment for later readers.

Thanks, yes, I will do this later today.